src/service_config/contrib_portal_config.py

```diff
@@ -0,0 +1,14 @@
+from service import orderly_volume_name, template_volume_name
+from docker_helpers import copy_between_volumes
+
+
+def configure_contrib_portal(template_report_version):
+    print("Configuring contrib portal")
+    path_to_templates = "archive/internal-2017-burden-estimates-template/{}/*.csv".format(template_report_version)
```

Comment:
I'm not sure about this bit here:

* Perhaps we should do nothing if the template_report_version is the empty string - this must be needed on any instance without the orderly volume from backup?
* What perhaps make the format `<reportname>/<id>` so that this year's templates are not hard coded into the deploy
* We discussed multiple templates - does this report have to generate all types now?